lib/mysql_framework/scripts/base.rb

```diff
@@ -50,6 +50,21 @@ def index_exists?(client, table_name, index_name)
         result.count >= 1
       end
 
+      def index_up_to_date?(client, table_name, index_name, columns)
+        result = client.query(<<~SQL)
```

Comment:
Is this structure correct (closing the parenthesis afer SQL)?

Reply:
Yes, see other SQL around in that file. A bit of background can be found here: https://medium.com/rubycademy/heredocs-tokens-as-method-argument-8fe3ed723d96